core/frontend/src/render/webgl/TechniqueFlags.ts

```diff
@@ -43,10 +43,13 @@ export const enum IsThematic { No, Yes }
  * @internal
  */
 export class TechniqueFlags {
+  /** System will set this to true upon initialization if necessary. */
+  public static requireEdgeTest = false;
```

Comment:
@mgooding @DStradley I'm not in love with this solution but I wanted to ensure we always apply the edge test (force surface discard) without having to add conditionals all over the place.